AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -687,9 +689,9 @@ def global_skylight(property: str = 'name') -> str:
         """
         properties = {
             'name': 'Global Skylight (IBL)',
-            'Diffuse Image': 'Controller|Configuration|Diffuse Image',
-            'Specular Image': 'Controller|Configuration|Specular Image',
-            'Exposure': 'Controller|Configuration|Exposure',
+            'Diffuse Image': 'Diffuse Image',
+            'Specular Image': 'Specular Image',
+            'Exposure': 'Exposure',
```

Comment:
do these actually have no property path? this is also Global Skylight (IBL) is that part of directional light?

Reply:
This was not changed by my code, and has nothing to do with my PR